### ISSUE TITLE
Screen Reader: Many Pages: The current menu link is not announced as such

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74864]
## Action Performed
1. Using Windows + Chrome, open the page https://www.expensify.com/terms
2. Navigate using TAB through the Terms of Service menu options
3. Observe the announcement for the currently selected menu link

Other occurrences:
1. On Privacy Policy

## Expected Result
The visually selected menu option should be announced as selected.

-"Terms of Service, current, link"

## Actual Result
As the user navigates through the Terms of Service menu buttons, the currently selected button, in this instance, the 'Terms of Service' is visually indicated as selected with a green bar and bold font, however it is only announced with the name of the button, and no context as to whether or not the button has already been selected or not.

- 'Terms of Service, link'

<li><a href="/terms" class="current">Terms of Service</a></li>

## Area issue was found
Terms of Service

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Use the aria-current to announce the current menu link as such.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1917" height="996" alt="Image" src="https://github.com/user-attachments/assets/d7ab36d0-b51d-44ed-b3da-90f3b327ca08" />

https://github.com/user-attachments/assets/362f10e4-b329-4cd1-a27f-53a9b1fc958e

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6993962
**Issue reported by:** NA
**Slack conversation:** NA
